packages/cosmos/src/error.rs

```diff
@@ -446,7 +446,7 @@ impl QueryErrorDetails {
                     // number.
                     CosmosSdkError::TxInMempool => NetworkIssue,
                     // Similar for account sequence errors
-                    CosmosSdkError::IncorrectAccountSequence => NetworkIssue,
```

Comment:
Comment above would need to be modified.

Reply:
Thanks, fixed!